### PR TITLE
CI: Enable unit-tests in CI

### PR DESCRIPTION
**Pull request by xaionaro**
The failing auto-test is fixed in https://github.com/facebookincubator/go-belt/pull/13

### CODE REVIEW

**Comment by u1f35c** [comment]
Looks reasonable, but holding off on approving until #13 lands and the unit tests all pass.

**Comment by u1f35c** [comment]
#13 has landed, can you rebase to confirm the tests all pass and then I'll approve?

**Comment by xaionaro** [comment]
> #13 has landed, can you rebase to confirm the tests all pass and then I'll approve?

@u1f35c : Done, sorry for the delay :)

**Review by u1f35c — APPROVED** [review]

**Pull request merged**
